lib/kovid.rb

```diff
@@ -3,6 +3,7 @@
 require 'kovid/version'
 require 'kovid/request'
 
+# Default namespace
```

Comment:
I don't think this makes sense. "Default namespace" doesn't say much. Perhaps you can switch this cop off?

Reply:
Ok, will check to disable the cop for this class and similar classes. :+1: